ext/zip/tests/utils.inc

```diff
@@ -1,4 +1,11 @@
 <?php
+function dump_entries_name($z) {
+	for($i=0; $i<$z->numFiles; $i++) {
+	    $sb = $z->statIndex($i);
+	    echo $i . ' ' . $sb['name'] . "\n";
+	}
+}
+
```

Comment:
Re-add this helper, don't know why was removed, 
it gives more better diff than the new one